packages/spatial/src/transform/components/TransformComponent.ts

```diff
@@ -71,7 +74,10 @@ export const TransformComponent = defineComponent({
         new Vector3(1, 1, 1)
       ) as Vector3,
       matrix: new Matrix4(),
-      matrixWorld: new Matrix4()
+      matrixWorld: new Matrix4(),
+      forward: () => TransformComponent.forward(entity),
```

Comment:
this is an anti pattern. component definitions should just store data, not any utility functions

Reply:
removed per our discussion